docs/learn/expert-guides/utilize-ipfs-storage.md

```diff
@@ -84,6 +84,14 @@ const url = await provider.upload(file);
 console.log('File URL:', url);
 ```
 
+**Using Cascade**
+
+```js
+import { CascadeUploader } from '@lukso/data-provider-cascade';
```

Comment:
I can't find that lib on NPM?

https://www.npmjs.com/search?q=%40lukso%2Fdata-provider-cascade

Reply:
Related to: https://github.com/lukso-network/tools-data-providers/pull/12